pandas/io/json/json.py

```diff
@@ -787,6 +791,10 @@ def _parse_no_numpy(self):
                 loads(json, precise_float=self.precise_float))}
             self.check_keys_split(decoded)
             self.obj = Series(dtype=None, **decoded)
+        elif orient == "records":
```

Comment:
Is the point of these updates to parse back into a Series? I'm not sure if that's entirely possible as it would be ambiguous whether a Series or DataFrame should be returned. At the very least would need a test case to demonstrate this though

Reply:
yeah, i did this just because in one pytest, it `read_json` from the output of `to_json` method, and then since previously, `series.to_json(orient='records')` returns a list of values, like `[1, 2, 3]`, and now the output of `to_json` turns to a list of dictionaries, like `[{'a': 1}, {'a': 2}, {'a': 3}]`, and thus if i didn't make this change, one pytest will fail, since this `read_json` also needs to be updated correspondingly.